mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -49,6 +50,8 @@ CLASS none/aeq net/minecraft/item/ItemStack
 		ARG 0 amount
 	METHOD g subtractStackSize (I)V
 		ARG 0 amount
+	METHOD i getMetadata ()I
+	METHOD j getMetadata2 ()I
```

Comment:
Nonononono. I'm pretty sure they're not both metadata. Could you look at the calls?